prefect_aws/credentials.py

```diff
@@ -67,3 +67,68 @@ def get_boto3_session(self) -> boto3.Session:
             profile_name=self.profile_name,
             region_name=self.region_name,
         )
+
+
+class MinIOCredentials(Block):
+    """
+    Block used to manage authentication with MinIO. Refer to the
+    [MinIO docs](https://docs.min.io/docs/minio-server-configuration-guide.html)
+    for more info about the possible credential configurations.
+
+    Args:
+        minio_root_user: Admin or root user.
+        minio_root_password: Admin or root password.
+        region_name: Location of server, e.g. "us-east-1".
+
+    Example:
+        Load stored MinIO credentials:
+        ```python
+        from prefect_aws import MinIOCredentials
+
+        minio_credentials_block = MinIOCredentials.load("BLOCK_NAME")
+        ```
+    """  # noqa E501
+
+    _logo_url = "https://www.socallinuxexpo.org/sites/default/files/logos/C2qJE7_XEAEn2MQ.png"  # noqa
```

Comment:
Actually I found a red bird here!
https://min.io/logo

```suggestion
    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/22vXcxsOrVeFrUwHfSoaeT/ee909d5309db9347858819cdbf5870b8/MINIO_Bird.png?h=250"  # noqa
```

And while I was searching, I realized we can just use an online tool to make white backgrounds transparent; for future reference:
https://onlinepngtools.com/create-transparent-png

Reply:
Oh, that's awesome! Would Alex use this tool when uploading, then?

Reply:
I recently got access granted for contentful so I added it to contentful already. I think you can also get access by filling out the request form too (just say need to implement block logos).

But yeah in the future, if there's a logo that's really hard to find a transparent background for, we can use this tool (whether it's for collections catalog or blocks logo)